STM32F405RGTx/Core/Src/main.cpp

```diff
@@ -133,6 +134,18 @@ int main(void)
   MX_TIM3_Init();
   /* USER CODE BEGIN 2 */
   mc_entry();
+
```

Comment:
remove these please, starting the PWM is something that should be done in the state machine I think